src/core/Emitter.js

```diff
@@ -23,17 +23,15 @@ var Emitter = {
                 this.on(key, value);
             }, this);
         } else {
-            var entry = this._eventTypes[type];
-            if (entry) {
-                var handlers = this._callbacks = this._callbacks || {};
-                handlers = handlers[type] = handlers[type] || [];
-                if (handlers.indexOf(func) === -1) { // Not added yet, add now.
-                    handlers.push(func);
-                    // See if this is the first handler that we're attaching,
-                    // and call install if defined.
-                    if (entry.install && handlers.length == 1)
-                        entry.install.call(this, type);
-                }
+            var entry = this._eventTypes[type],
```

Comment:
This assumes that _eventTypes exists, which is not true when there are no _events on class. This is case for example of Project class.